lib/improver/generate_ancillaries/generate_ancillary.py

```diff
@@ -140,7 +140,7 @@ def __repr__(self):
         return result
 
     @staticmethod
-    def sea_mask(landmask, orog_band):
+    def sea_mask(landmask, orog_band, sea_fillvalue=None):
         """
```

Comment:
Can we use "sea_fill_value" - fits better with naming conventions.

Reply:
Amended.